test/util/verifiers/serving_app.go

```diff
@@ -100,10 +100,6 @@ func (v verifySimpleWebApp) Verify(ctx context.Context, adminRESTConfig *rest.Co
 		currRoute, err := dynamicClient.Resource(gvr("route.openshift.io", "v1", "routes")).
 			Namespace(namespace.GetName()).Get(ctx, route.GetName(), metav1.GetOptions{})
 		if err != nil {
```

Comment:
usually i like if err != lastErr and yell then, you still get the sequence of unique errors on the way to failure